packages/endpoints-credential-types-registrar/src/init-server.js

```diff
@@ -44,12 +44,7 @@ const initServer = (server) => {
     .register(httpClientPlugin, {
       name: 'fetch',
       options: pick(
-        [
-          'nodeEnv',
-          'requestTimeout',
-          'traceIdHeader',
-          'useExistingGlobalAgent',
```

Comment:
the last review i said rename this flag `useGlobalDispatcher`. i dont understand why you removed it